src/API/src/db/shared/entities/reference.entity.ts

```diff
@@ -19,6 +20,10 @@ export class Reference extends BaseEntity {
   @Field({ nullable: true })
   journal_title: string;
 
+  @Column('varchar', { length: 500, nullable: true })
```

Comment:
I don't think we want this nullable - this is the field that we expect all reference entities to have